gui/app.py

```diff
@@ -83,7 +85,7 @@ def __init__(self, filenames):
         self.connect_signals()
 
         # Start the independent threads
-        self.video_thread.start()
```

Comment:
#45 renamed this to `video_thread` which seems to have gotten reverted in all the merge conflicts, so can you rename this back to `video_thread`?